### ISSUE TITLE
Fails to retrieve some packages after the switch to curl

### ISSUE THREAD

**Issue by terencode** [AladW/aurutils#706]
```
aur sync carla-bridges-win32-git

==> Using [aur] repository
Warning: <stdin>:1: warning: 'url' requires parameter
curl: no URL specified!
curl: try 'curl --help' for more information
```

I wonder if it happens because of the  '-' as mingw-w64-gcc also has the same problem.

When running `AUR_DEBUG=1 aur sync  "carla-bridges-win64-git"`
[aurutils.log](https://github.com/AladW/aurutils/files/4405772/aurutils.log)

**Comment by rafasc** [comment]
A shorter way to trigger this bug is simply: `aur query -t info </dev/null`

`aur sync` triggers this error via `aur-depends` that calls `aur-query` without input under certain conditions. See: https://github.com/AladW/aurutils/blob/master/lib/aur-depends#L46  
We either detect those cases in `depends` or make `query` gracefully exit on empty input.

**Comment by AladW** [comment]
Fixed with #707
